cli/flags.rs

```diff
@@ -1389,6 +1391,18 @@ fn no_check_arg<'a, 'b>() -> Arg<'a, 'b> {
     .help("Skip type checking modules")
 }
 
+fn no_check_certificate_arg<'a, 'b>() -> Arg<'a, 'b> {
+  Arg::with_name("no-check-certificate")
+    .long("no-check-certificate")
+    .min_values(0)
+    .takes_value(true)
+    .use_delimiter(true)
+    .require_equals(true)
+    .help("Allow network access")
+    .validator(crate::flags_allow_net::validator)
```

Comment:
Additional validation is required to allow only IP and domain names without port.